Blog on SIG Node spotlight 2021

dewan-ahmed opened the pull request:
This is a blog on SIG Node spotlight capturing my conversations with [Elana Hashman](https://twitter.com/ehashdn) & [Sergey Kanzhelev](https://twitter.com/SergeyKanzhelev).

reylejano commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,76 @@
+
+### Are there any particular skills you’d like to recruit for? What skills are contributors to SIG Usability likely to learn?
+
+SK: SIG Node works on many workstreams in very different areas. All of these areas are on system level. For the  typical code contributions you need to have a passion for building and utilizing low level APIs and writing performant and reliable components. Being a contributor you will learn how to debug and troubleshoot, profile, and monitor these components, as well as user workload that is run  ...
```
```suggestion
SK: SIG Node works on many workstreams in very different areas. All of these areas are on system level. For the typical code contributions you need to have a passion for building and utilizing low level APIs and writing performant and reliable components. Being a contributor you will learn how to debug and troubleshoot, profile, and monitor these components, as well as user workload that is run by these components. Often, with the limited to no access to Nodes, as they are running production workloads.
```
nit: extra space

reylejano commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,76 @@
+
+[Davanum Srinivas](https://twitter.com/dims) and I each ran a cohort of group mentoring to help teach new contributors the skills to become Node reviewers, and if there's interest we can work to find a mentor to run another session. I also encourage new contributors to attend our Node CI Subproject meeting: it's a smaller audience and we don't record the triage sessions, so it can be a less intim ...
+
+
```
```suggestion
```

reylejano commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,76 @@
+---
+layout: blog
+title: "Spotlight on SIG Node"
+date: 2021-08-13
```
```suggestion
date: 2021-09-06
```
Next available publish date is Sept 6

reylejano commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,76 @@
+
+Are you interested in learning about what [SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) does and how you can get involved? Well, you're at the right place. SIG Node is responsible for the components that support the controlled interactions between pods and host resources. In this blog, we have summarized our conversation with [Elana Hashman](https://twitter.com/ehashdn) ...
+
+
```
```suggestion
```
nit: remove extra line break

reylejano commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,76 @@
+### Could you tell us a little about what SIG Node does?
+
+SK: SIG Node is a vertical SIG responsible for the components that support the controlled interactions between the pods and host resources. We manage the lifecycle of pods that are scheduled to a node. This SIG's focus is to enable a broad set of workload types, including workloads with hardware specific or performance sensitive requirements. All while maintaining isolation boundaries between pod ...
+
+
```
```suggestion
```
nit: remove extra line breaks

reylejano commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,76 @@
+### Could you tell us a little about what SIG Node does?
+
+SK: SIG Node is a vertical SIG responsible for the components that support the controlled interactions between the pods and host resources. We manage the lifecycle of pods that are scheduled to a node. This SIG's focus is to enable a broad set of workload types, including workloads with hardware specific or performance sensitive requirements. All while maintaining isolation boundaries between pod ...
+
+
+
```
```suggestion

```
nit: remove extra line breaks

reylejano replied:
```suggestion
```
nit: remove extra line breaks

reylejano commented:
Hi @ehashman and @SergeyKanzhelev ,

Please review this blog PR as it's an interview with you about SIG Node. 
Please add comments or suggestions if needed or give an `/lgtm` if this interview is ok and ready to publish as a blog.

Thank you!

ehashman commented:
/assign @ehashman @SergeyKanzhelev 

suspect it will help if we actually get tagged on GitHub, I forgot about this :sweat_smile:

SergeyKanzhelev reviewed (approved):
/lgtm

thanks!

ehashman reviewed (commented):
Checked over my answers, they look good -- will let Sergey sign off on his.

Do we want to add links?

sftim commented:
Blog team, is this looking good to merge? If it is, let's pick a publication date (we can push an extra commit to set that).

nate-double-u commented:
I'd suggest that we wait for the author to change up the introduction before we merge.

reylejano commented:
The author, @dewan-ahmed , is out until Sept 17. 
@dewan-ahmed please take a look at @nate-double-u's [comment](https://github.com/kubernetes/website/pull/29380/files#r696125785) and @ehashman's [comment](https://github.com/kubernetes/website/pull/29380#discussion_r704661435) when you're back.

dewan-ahmed commented:
Thanks for the note @reylejano 

I'll address the above comments via another commit by EOD on Sept22.

sftim reviewed (commented):
Publication date is now in the past
/hold

as we need to reschedule (SIG Docs blog team can take care of this)

dewan-ahmed commented:
Hi @reylejano @sftim I addressed @nate-double-u's [comment](https://github.com/kubernetes/website/pull/29380/files#r696125785) and @ehashman's [comment](https://github.com/kubernetes/website/pull/29380#discussion_r704661435). I also adjusted the date to Sept27, 2021 as a potential publication date.

Please let me know if you see any other fix required.

reylejano commented:
Thank you for the updates
Re-applying lgtm
/lgtm

reylejano commented:
Preview of blog: https://deploy-preview-29380--kubernetes-io-main-staging.netlify.app/blog/2021/09/27/sig-node-spotlight-2021/

@dewan-ahmed thank you for addressing the comments
/approve
/hold cancel

nate-double-u commented on content/en/blog/_posts/2021-08-13-SIG-Node-Spotlight/index.md:
```diff
@@ -0,0 +1,73 @@
+
+## Introduction
+
+[SIG Node](https://github.com/kubernetes/community/tree/master/sig-node) is responsible for the components that support the controlled interactions between pods and host resources. The work of this group manages the lifecycle of pods that are scheduled to a node, and focuses on enabling a broad set of workload types, including workloads with hardware specific or performance sensitive requirements ...
```
This is a better introduction, but now it's mostly a copy of the answer to the first question so it's giving away some of the post's content early.

Since this PR has been merged, but the publication date is still in the future, I suggest opening a follow up PR to rewrite this introduction.

dewan-ahmed replied:
Good catch @nate-double-u 

I didn't notice that the same was reused in the first question. I'm working on rewriting the intro and expect to create another PR by end of the day.

dewan-ahmed replied:
@nate-double-u https://github.com/kubernetes/website/pull/29787

nate-double-u reviewed (commented):
Thanks for updating the introduction, but I've still got some comments about it:
